.github/workflows/ci.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
```

Comment:
Why not just use v4 ?

Reply:
I am not quite sure, but I know this hash is for v4.1.1.

Reply:
I'm guessing that's how renovate has now decided to do it - because after all, tags can change which could randomly change the outcome.